docs/docs/database/settings.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: 'Settings'
+sidebar_position: 3
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+Below you can find the official schema (cue) and an example to configure your postgres database:
+
+<Tabs groupId="package-manager">
+  <TabItem value="schema" label="schema" default>
+
+```cue
```

Comment:
our users and cue... I would either omit cue altogether or show TOML by default (to the left)

Reply:
I will change the default tab for now, we can decide later on. Ideally we'd render some nice docs from the schema but haven't had the time to research how to do that yet.